lesson28/script.js

```diff
@@ -0,0 +1,38 @@
+//1.Реализовать класс, описывающий окружность. В классе должны быть следующие компоненты:
+
+// поле, хранящее радиус окружности;
+// get-свойство, возвращающее радиус окружности;
+// set-свойство, устанавливающее радиус окружности;
+// get-свойство, возвращающее диаметр окружности;
+// метод, вычисляющий площадь окружности;
+// метод, вычисляющий длину окружности.
+
+class SomeClass {
+  constructor(x) {
+    this.x = x;
+  }
```

Comment:
можно назвать так, чтоб было понятнее
```suggestion
  constructor(radius) {
    this._radius = radius;
  }
```